tests/lib/rules/no-console.js

```diff
@@ -40,24 +40,39 @@ ruleTester.run("no-console", rule, {
     invalid: [
 
         // no options
-        { code: "console.log(foo)", errors: [{ messageId: "unexpected", type: "MemberExpression" }] },
-        { code: "console.error(foo)", errors: [{ messageId: "unexpected", type: "MemberExpression" }] },
-        { code: "console.info(foo)", errors: [{ messageId: "unexpected", type: "MemberExpression" }] },
-        { code: "console.warn(foo)", errors: [{ messageId: "unexpected", type: "MemberExpression" }] },
+        { code: "console.log(foo)", errors: [{ messageId: "unexpected", type: "MemberExpression", suggestions: [{ messageId: "removeConsole", data: { propertyName: "log" }, output: "" }] }] },
```

Comment:
Can you break these out into multiple lines to make them easier to review? I know the original had these on single lines, but adding in suggestions makes these even harder to read on a single line.